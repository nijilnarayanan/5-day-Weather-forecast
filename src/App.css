body {
  background-color: #0D4273;
}

.App {
  padding: 75px 0px;
}

@media screen and (min-width: 767px) {
  .App {
    max-width: 790px;
    display: flex;
    flex-direction: column;
    margin: 0 auto;
    padding: 150px 0px;
  }
}