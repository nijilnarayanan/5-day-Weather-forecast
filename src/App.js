import React from 'react'
import Header from './Header'
import WeatherContainer from './WeatherContainer'
import Band from './Band'
import './App.css'

function App() {
	return (
		<div className='App'>
			<Header />
			<WeatherContainer />
			<Band />
		</div>
	)
}

export default App
