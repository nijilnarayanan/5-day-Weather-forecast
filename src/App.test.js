import React from 'react'
import { mount } from 'enzyme'
import App from './App'

test('Run Test <App />', () => {
	const component = mount(<App />)
	expect(component.debug()).toMatchSnapshot()
})
