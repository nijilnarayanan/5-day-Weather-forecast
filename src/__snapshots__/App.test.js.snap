// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Run Test <App /> 1`] = `
"<App>
  <div className=\\"App\\">
    <Header>
      <styled.section>
        <section className=\\"sc-bdfBwQ jEWXLh\\">
          <styled.div>
            <div className=\\"sc-gsTCUz jLFiED\\">
              5 Day Weather Forecast
            </div>
          </styled.div>
          <styled.div>
            <div className=\\"sc-dlfnbm fGPqXe\\">
              <styled.img src=\\"location.png\\">
                <img src=\\"location.png\\" className=\\"sc-hKgILt gqiLmC\\" />
              </styled.img>
              <styled.span>
                <span className=\\"sc-eCssSg ldyRwL\\">
                  Mai
                </span>
              </styled.span>
            </div>
          </styled.div>
        </section>
      </styled.section>
    </Header>
    <WeatherContainer>
      <styled.section>
        <section className=\\"sc-kEjbxe bXnKhx\\" />
      </styled.section>
    </WeatherContainer>
    <Band>
      <styled.section>
        <section className=\\"sc-iqHYGH fEzGXk\\">
          <styled.div>
            <div className=\\"sc-crrsfI duZmFZ\\">
              Made with ❤️ by Nijil Narayanan
            </div>
          </styled.div>
        </section>
      </styled.section>
    </Band>
  </div>
</App>"
`;
